src/features/projects/components/MapboxMap.tsx

```diff
@@ -362,7 +416,7 @@ export default function MapboxMap({
         mapboxApiAccessToken={mapboxToken}
```

Comment:
I see all network requests changed from mapbox to arcgis, so I wonder if the requests still count on our mapbox account as the token is still configured here?